.github/workflows/fast_testing.yml

```diff
@@ -18,18 +18,40 @@ jobs:
       fail-fast: false
       matrix:
         tarantool: ['1.10', '2.8']
+        icu: ['release-70-1/icu4c-70_1', 'release-71-1/icu4c-71_1']
 
     runs-on: ubuntu-latest
     steps:
+      - name: Install libicu
+        run: |
+          cd ~
+          sudo apt remove libicu-dev libicu66 nginx
+          sudo apt install python3
+          wget https://github.com/unicode-org/icu/releases/download/${{ matrix.icu }}-src.tgz
+          tar zxvf icu4c-*.tgz
+          cd icu/source
+          ./runConfigureICU Linux
+          make -j $(nproc)
+          sudo make install
+          sudo ldconfig -v
+
       - name: Install tarantool ${{ matrix.tarantool }}
-        uses: tarantool/setup-tarantool@v1
-        with:
-          tarantool-version: ${{ matrix.tarantool }}
+        run: |
+          cd ~
+          git clone https://github.com/tarantool/tarantool.git
+          cd tarantool
+          git checkout ${{ matrix.tarantool }}
+          cmake -DENABLE_DIST=ON .
```

Comment:
Nit: It is better to disable feedback daemon to don't pollute its statistics.

Reply:
OTOH, we have it in usual builds, so it would not change the picture much. Nevermind.

Reply:
> Nit: It is better to disable feedback daemon to don't pollute its statistics.

No problem. Fixed.